src/utils/rates.js

```diff
@@ -1,4 +1,2 @@
-export const getExchangeRatesForCurrencies = (rates, base, quote) => {
-	if (!rates) return 0;
-	return rates[base] * (1 / rates[quote]);
-};
+export const getExchangeRatesForCurrencies = (rates, base, quote) =>
+	rates === null ? 0 : rates[base] * (1 / rates[quote]);
```

Comment:
the strict null check is intentional? (I usually do `==` just in case `rates` is undefined).
`==` will match both null/undefined
`===` matches only null... so potentially it can break?

Reply:
hmmm knowing that it can only be null as the defaultState sets it I'd probably say we keep it strict?

Reply:
yeah sure - if it was intentional its ok! 👍